_pages/coc.md

```diff
@@ -3,3 +3,48 @@ layout: page
+- Bring a positive attitude to all interactions.
+- Assume that everyone else is being light-hearted and acting in good faith unless demonstrated otherwise.
+
+## Conflict Resolution
+If you or any other member is being treated in a way that violates this CoC, please contact an administrator immediately. You can contact us via slack, the [contact form](/contact/) on this site, or by [email](mailto:admin@devanooga.com).
+Administrators will attempt to resolve the conflict in a manner that is accessible to the rest of the devanooga community, such as in a public channel. If open dialogue puts the privacy of one or more of the parties at risk, parts or all of the conflict resolution may be carried out in private.
+
+## Conduct of Administrators
+devanooga is organized by Chattanooga-area members. We want to foster a community that is driven and shaped by the members. With that in mind:
+
+- Administrators never act alone. Every action taken by an administrator is discussed with the larger group of admins before the action is taken.
+- Administrators will do everything within reason to act transparently. In some cases, it may be necessary (for the privacy of members) to do things "behind closed doors". We will avoid this whenever possible.
+- Administrators can be removed from power. If they violate the CoC, or otherwise intentionally prevent the growth of the community, they can be removed by a majority vote of the other administrators.
+
+## Changes to the Code of Conduct
+This is a living document. If you would like to openly discuss changes to this CoC, we welcome you to do any of the following:
+- Start a dialogue with administrators and the community in the `#general` Slack channel.
+- Use [our contact form](/contact) to send a message directly to administrators.
+- Send administrators an email at <admin@devanooga.com>.
+- Start an issue on [this website’s Github repository](https://github.com/devanooga/devanooga.github.io).
+- Fork this website’s Github repository, commit your changes, and issue a Pull Request.
+
+## Use of the Slack Organization
+The use of the devanooga Slack Organization is a privilege we provide and maintain for users that are in good standing with the community. At any time, the admins may choose to revoke the privileges of a user to access the Slack Organization.
+
+We ask that users keep these additional rules in mind when using the devanooga Slack Organization.
+
+- Users are allowed only one account.
+- Users may not impersonate other users, celebrities, etc.
+- Try to find a relevant channel for a topic of discussion. For example, don't talk about your latest trip to the gun range in `#ruby`.
+- Political, religious, and otherwise sensitive or controversial discussions are welcome, but we ask that users keep such topics out of the #general and #random channels.
```

Comment:
Honestly I feel that this rule should be that these topics are _kept_ to their relevant channels (eg: #politics or #religion), but we don't have those channels yet. I would add that if someone feels your topic is better suited for #politics or #religion that generally that opinion should be respected _even moreso than you would when this suggestion is about other topics_ (eg: "this isn't #ruby it's really better for #devops") . You can easily get political on all kinds of subjects so the ability to "bleed over" (eg: almost anything tying to recent events can be political) while "staying on topic" is possible, but honestly I'd like to see that not happen.

Though I'm fine with addressing this when it happens and talking to the community about it when the time comes (if someone spins up a #politics though I'll recommend amending this).